WMF Framework/Remote Notifications/RemoteNotificationsOperationsController.swift

```diff
@@ -102,6 +97,17 @@ class RemoteNotificationsOperationsController: NSObject {
             self.operationQueue.addOperations(operations + [completionOperation], waitUntilFinished: false)
         })
     }
+    
+    private func cookieDomainForLanguageCode(_ languageCode: String) -> String {
+        switch languageCode {
+        case "wikidata":
+            return Configuration.current.wikidataCookieDomain
+        case "commons":
```

Comment:
Great point, I pushed an update with a new RemoteNotificationProject enum since I found I was using these types of identifiers often across classes. Felt cleaner than a string constant but let me know if it feels like a bit much or poorly named.

Reply:
Looks awesome, thanks!